code/obj/artifacts/artifactprocs.dm

```diff
@@ -263,7 +263,11 @@
 						for(var/i=1,i<rand(1,3),i++)
 							src.ArtifactDevelopFault(100)
 				else if (A.activated)
-					src.ArtifactDeactivated()
+					if (istype(src, /obj/item/artifact/bag_of_holding))
+						if (!user || user.a_intent == INTENT_HARM)
+							src.ArtifactDeactivated()
```

Comment:
Intent-based interactions kinda suck. Would rather just have deactivators not be storable at all in bags of holding that they'd deactivate

Reply:
Sure. Alternate idea, maybe allow them to be added by clicking the activator key with the bag of holding?

Reply:
comedy option: let it go in but then detonate the bag a la putting a wand of cancellation in a bag of holding in nethack 😈

Reply:
I went with tying it into the existing bag of holding negative effects, so adding a bag of holding or an activator key will do a bad effect